packages/starknet-snap/src/ui/fragments/index.tsx

```diff
@@ -0,0 +1,2 @@
+export * from './RowUI';
+export * from './JsonDataUI';
```

Comment:
why this 2 components is in fragments folder

Reply:
I wanted to have small components in the fragments folder and big components representing full page in the components folder. wdyt ?

Reply:
yea, but why address UI and network Ui and others are in components, they are not big

Reply:
this is fixed in next PR